kratos/includes/variables.h

```diff
@@ -180,6 +181,12 @@ namespace Kratos
     KRATOS_DEFINE_VARIABLE( double, CHARACTERISTIC_LENGTH_MULTIPLIER )
 
     //AdjointFluidApplication
+    KRATOS_DEFINE_3D_VARIABLE_WITH_COMPONENTS( ADJOINT_VECTOR_1 )
+    KRATOS_DEFINE_3D_VARIABLE_WITH_COMPONENTS( ADJOINT_VECTOR_2 )
+    KRATOS_DEFINE_3D_VARIABLE_WITH_COMPONENTS( ADJOINT_VECTOR_3 )
+    KRATOS_DEFINE_3D_VARIABLE_WITH_COMPONENTS( AUX_ADJOINT_VECTOR_1 )
```

Comment:
Do you anticipate that we will need to distinguish between the fluid side adjoints and structure side adjoints? If so, these are not very descriptive.

Reply:
I decided here to add some generic adjoint variables. I'm still not sure what the best strategy will be when we start mixing elements.